src/rpc/parameter_conversion.cpp

```diff
@@ -173,8 +173,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "filtertransactions", 0, "options" },
     { "filteraddresses", 0, "offset" },
     { "filteraddresses", 1, "count" },
-    { "filteraddresses", 2, "sort_code" },
-    { "filteraddresses", 4, "match_owned" },
+    { "filteraddresses", 3, "sort_code" },
+    { "filteraddresses", 5, "match_owned" },
```

Comment:
don't we specify here an index (2) for the `"sort_key"` parameter as well?

Reply:
This table only holds the arguments that need to be converted from strings to other JSON types, and `sort_key` needs to remain a string.